contracts/linkdrop/src/create_user_account.rs

```diff
@@ -1,13 +1,15 @@
 use crate::*;
 use near_sdk::{AccountId, Gas, PublicKey};
 
+const BASE_GAS: Gas = Gas(25_000_000_000_000); // 25 TGas
```

Comment:
What do you think about instead of having a default gas and multiply it, create a struct like `Tgas(25)` or a  function `to_tgas(50)`?

Reply:
please look at this:
https://github.com/NEAR-labs/contracts.near-linkdrop/pull/13/commits/69a3fae0967aedab6762d1dde07d4662f5a69d06